guides/common/modules/proc_configuring-ipxe-environment.adoc

```diff
@@ -0,0 +1,55 @@
+[id="Configuring_iPXE_Environment_{context}"]
+= Configuring iPXE environment
+
+Configure an iPXE environment on all {SmartProxies} that you want to use for iPXE provisioning.
+
+ifndef::foreman-deb[]
+.HTTP security consideration
+In {EL}, security-related features of iPXE are not supported and the iPXE binary is built without security features.
+For this reason, you can only use HTTP but not HTTPS.
+ifdef::satellite[]
+For more information, see https://access.redhat.com/solutions/3483601[{RHEL} HTTPS support in iPXE].
+endif::[]
+ifndef::satellite[]
+You can recompile iPXE from source to use security features such as HTTPS.
+endif::[]
+endif::[]
+
+.Prerequisites
+include::snip_prerequisite-configured-smart-proxy-registration-provisioning.adoc[]
+
+.Procedure
+. Enable the *TFTP* and *HTTPboot* services on your {SmartProxy}:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} \
+--foreman-proxy-httpboot true \
+--foreman-proxy-tftp true
+----
+. Install the `{ipxe-package}` package on your {SmartProxy}:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {project-package-install} {ipxe-package}
+----
+. Copy iPXE firmware to the TFTP directory.
```

Comment:
```suggestion
. Copy the iPXE firmware to the TFTP directory.
```

Reply:
Nope. I'm mentioning the firmware for the first time in this module + it's uncountable.